src/realtime/webrtc-connection.ts

```diff
@@ -128,6 +99,18 @@ export class WebRTCConnection {
 				case "ice-candidate":
 					if (msg.candidate) await this.pc.addIceCandidate(msg.candidate);
 					break;
+				case "ice-restart":
+					const turnConfig = msg.turn_config &&
+						typeof msg.turn_config === 'object' &&
+						'username' in msg.turn_config &&
+						'credential' in msg.turn_config &&
+						'server_url' in msg.turn_config
+						? msg.turn_config as TurnConfig
+						: undefined;
+					if (turnConfig) {
+						await this.setupNewPeerConnection(turnConfig);
+					}
```

Comment:
since we're controlling the backend I think we can safely simplify it to:

```suggestion
					const turnConfig = msg.turn_config;
					if (turnConfig) {
						await this.setupNewPeerConnection(turnConfig);
					}
```